go/client/chat_cli_utils.go

```diff
@@ -125,6 +126,9 @@ func parseConversationResolver(ctx *cli.Context, tlfName string) (resolver conve
 	if resolver.TopicType, err = parseConversationTopicType(ctx); err != nil {
 		return resolver, err
 	}
+	if resolver.TopicType == chat1.TopicType_CHAT && len(resolver.TopicName) != 0 {
+		return resolver, errors.New("multiple topic names are not yet supportted with chat conversations")
```

Comment:
`keybase chat send --topic-name test hi` tries to insert a message to set topic name which seems broken for now (but worked at some point I think). `keybase chat read --topic-name test` would simply resolve to the default conversation without any topic name, which is not broken but also can be a little be confusing. Doing this before hitting server gives user clear message (without RTT delays) that this won't work, which is kind of nice :-)

I'll fix the error message; Thanks!